black: adjust example descriptions for `--diff` and `--check` examples (#9211)
tldr-pages/tldr: pages/common/black.md

`black -c "{{code}}"`

- Output whether a file or a directory would have changes made to them if they were to be formatted:

`black --check {{path/to/file_or_directory}}`

> - Output any changes that would be made to a file or a directory if they were to be formatted:

Reviewer: Is this a dry-run?
```suggestion
- Output changes that would be made to a file or a directory without performing them (dry-run):
```
